contracts/DCAHubCompanion/DCAHubCompanionHubProxyHandler.sol

```diff
@@ -32,13 +32,28 @@ abstract contract DCAHubCompanionHubProxyHandler is IDCAHubCompanionHubProxyHand
     address _owner,
     IDCAPermissionManager.PermissionSet[] calldata _permissions,
     bytes calldata _miscellaneous
-  ) external payable returns (uint256 _positionId) {
+  ) public payable virtual returns (uint256 _positionId) {
     _approveHub(address(_from), _hub, _amount);
     _positionId = _miscellaneous.length > 0
       ? _hub.deposit(_from, _to, _amount, _amountOfSwaps, _swapInterval, _owner, _permissions, _miscellaneous)
       : _hub.deposit(_from, _to, _amount, _amountOfSwaps, _swapInterval, _owner, _permissions);
   }
 
+  /// @inheritdoc IDCAHubCompanionHubProxyHandler
+  function depositWithAllBalanceProxy(
```

Comment:
Do we need to keep adding `Proxy` on the functions name? I think we can take 'em out, right?

Reply:
All of them, or just the new ones?

Reply:
I'd take the `Proxy` suffix out of all the functions.

Reply:
Will do it in #105